### PR TITLE
change output model to _tocRel

### PR DESCRIPTION
**Pull request by OsmondJiang**
fix bug: https://dev.azure.com/ceapex/Engineering/_workitems/edit/94425

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/dotnet/docfx/pull/4787)

### CODE REVIEW

**Comment by OsmondJiang** [comment]
diff are expected

File: src/docfx/build/page/BuildPage.cs [hunk 1]
```diff
@@ -93,7 +93,7 @@ public static async Task<IEnumerable<Error>> Build(Context context, Document fil
             }
 
             outputMetadata.Locale = file.Docset.Locale;
-            outputMetadata.TocRel = tocMap.FindTocRelativePath(file);
+            outputMetadata.TocRel = !string.IsNullOrEmpty(inputMetadata.TocRel) ? inputMetadata.TocRel : tocMap.FindTocRelativePath(file);
```
Comment (yufeih) [inline src/docfx/build/page/BuildPage.cs]: So for this type of archived content, it does not need TOC map. So TOC map build can be lazily evaluated.

**Comment by OsmondJiang** [comment]
> Instead of changing docfx, is it better to change `docs-ui` ?

Change in template need confirmation with template team, and also we already have '_op_contributionInfo' output name which is also ugly as this, so let's do this naming thing together later.

File: src/docfx/build/page/BuildPage.cs (hunk 1 above)
Comment (OsmondJiang) [reply src/docfx/build/page/BuildPage.cs]: We don't know if all files in archived content has "_tocRel" metadata, but you are right, we can lazy the toc map. I prefer to do it in another PR.

Comment (yufeih) [reply src/docfx/build/page/BuildPage.cs]: yep, just bring it up.

File: docs/specs/legacy.yml [hunk 2]
```diff
@@ -62,7 +62,6 @@ outputs:
         <meta name=\"search.ms_product\" content=\"\" />
         <meta name=\"search.ms_sitename\" content=\"Docs\" />
         <meta name=\"site_name\" content=\"Docs\" />
-        <meta name=\"toc_rel\" content=\"toc.json\" />
```
Comment (superyyrrzz) [inline docs/specs/legacy.yml]: We still need `toc_rel` in output page meta. Does it still work?

Comment (OsmondJiang) [reply docs/specs/legacy.yml]: That's template model output, not covered by our spec test now. It's converted by integration test, check the test result

**Pull request merged**